### ISSUE TITLE
block -serial

### ISSUE THREAD

**Issue by Maharajacentrado** [kreativhut/z-blockly-tracker#270]
<img width="116" alt="image" src="https://user-images.githubusercontent.com/104857726/181665761-c023ccf1-af9b-42c0-ad0c-b1d4bbdd610f.png">

Add the input to enter the data like the picture below
<img width="212" alt="image" src="https://user-images.githubusercontent.com/104857726/181665919-9ee9f530-2249-4c33-9d8c-3946bb7d45a1.png">

**Comment by kreativhut** [comment]
Could you elaborate the requirement here @Maharajacentrado

**Comment by Maharajacentrado** [comment]
for the above "myserial same" block add an input , whatever we are typing inside inside it should appear in the native code 

if we type hello inside the input area, 

mySerial.print("Hello"); //for myserial same 
mySerial.println("Hello"); // for myserial new

**Comment by kreativhut** [comment]
is it a new Block? @Maharajacentrado

**Comment by Maharajacentrado** [comment]
No, already we have a block named as "myserial same" under serial block section... we have to add the input area as we have in the serial print on new line block.
